cmd/operator-sdk/bundle/create.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/spf13/pflag"
 )
 
-type bundleCreateCmd struct {
+type createCmd struct {
```

Comment:
I agree that the name is redundant and we could/should change but it is not part of the scope of this PR. 
Could we do a PR just to do the renames?

Reply:
woops, forgot to revert this when I moved everything back to private. fixed.